.github/workflows/web.yml

```diff
@@ -5,11 +5,11 @@ jobs:
   test_web:
       runs-on: ubuntu-latest
       steps:
-        - uses: actions/checkout@v2
-        - name: Set up Python 3.8.1
-          uses: actions/setup-python@v2
+        - uses: actions/checkout@v3
+        - name: Set up Python 3.10.9
```

Comment:
Not sure we need python here at all since everything in the build step is dockerised.